webapp/src/sui.tsx

```diff
@@ -1212,6 +1225,18 @@ export class Modal extends data.Component<ModalProps, ModalState> {
 
     componentWillUnmount() {
         cancelAnimationFrame(this.animationRequestId);
+        if (!this.props.dontRestoreFocus) {
+            let toFocus = this.state.previouslyFocused as HTMLElement;
+            while (toFocus && !this.isFocusable(toFocus)) {
```

Comment:
You could refactor both this parent-finding logic and isFocusable into utils.tsx in react-common and import it here and in the focus trap

Reply:
Okay, cool. I wasn't certain about putting importing something from react-common into here, so that's why I made the duplicate in `sui.tsx`, I'll move and import.